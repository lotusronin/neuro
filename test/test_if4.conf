conf = {
'src' : 'test_if4.nro',
'expected' : 'test_if4.expected'
}
