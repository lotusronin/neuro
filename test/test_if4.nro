import stdio;

fn main() : u32 {
    var1 := 4;
    var2 := 5;
    if(var1 < var2)
        stdio::puts("var2 is greater!");
    else
        stdio::puts("var1 is greater!");
    return 0;
}
