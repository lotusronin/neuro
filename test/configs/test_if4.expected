var2 is greater!
